configure.ac

```diff
@@ -1316,6 +1316,14 @@ if test x$build_defi_wallet$build_defi_cli$build_defi_tx$build_defid$use_bench$u
   AC_MSG_ERROR([No targets! Please specify at least one of: --with-utils --with-libs --with-daemon --enable-bench or --enable-tests])
 fi
 
+AC_CHECK_PROG([local_rust_toolchain],cargo,yes)
```

Comment:
I'm not sure if we need this config, rather than use CARGO_HOME and RUSTUP_HOME to implicitly indicate it and work intuitively than an extra config

Reply:
Removed config and check.